ASM/c/invertYaxis.c

```diff
@@ -0,0 +1,21 @@
+#include "invertYaxis.h"
+
+extern uint8_t CFG_INVERT_YAXIS_IN_FIRST_PERSON_CAMERA;
+
+void manage_invert_yaxis() {
+
+    if (CFG_INVERT_YAXIS_IN_FIRST_PERSON_CAMERA) {
+        // Never if a choice textbox is on screen.
+        if (z64_MessageGetState(((uint8_t *)(&z64_game)) + 0x20D8) == 4) {
+            return;
+        }
+        if (z64_game.camera_mode == 0x06 ||
+            z64_game.camera_mode == 0x07 ||
+            z64_game.camera_mode == 0x0A ||
+            z64_game.camera_mode == 0x0B) {
```

Comment:
Might seem a little hypocritical considering how many unexplained "magic numbers" we have like this in the codebase, but I'd appreciate a comment labelling each of the states here so I can see why these are being checked for

Reply:
No you're right, it was lazy on my part.
I added the whole enum from decomp to make the values much clearer.